### DISCUSSION TITLE
BK3266 support

### DISCUSSION THREAD

**Discussion by Anesred** [tomaskovacik/vwcdavr#19] [Q&A] [answered]
Hi, does this code (specifically cdc_arduino_ported_k9spud_vwcdpic_with_BK3254.ino) support BK3266 aswell? and does it support all BK3266 or only HY5066 HYT?

**Comment by tomaskovacik** [comment, marked as answer]
Hi it depends on what AT commands it is using, but I am using it HYT version,

**Reply by Anesred** [reply]
I have one more question, what is the correct pinout for that code? I'm not too sure where to connect rx/tx of the bt module and the command, data and clock from the headunit

**Comment by tomaskovacik** [comment]
it depends on version of micro you are using, refer to these defines:


#if defined(__AVR_ATmega88__) || defined(__AVR_ATmega88A__) || defined(__AVR_ATmega328__) || defined(__AVR_ATmega328A__) || defined(__AVR_ATmega328P__) || defined(__AVR_ATmega328PA__) || defined(__AVR_ATmega328PB__)
#define RADIO_COMMAND      PB0 //ICP
#define RADIO_COMMAND_DDR  DDRB
#define RADIO_COMMAND_PORT  PORTB
#define RADIO_COMMAND_PIN PINB
#define RADIO_CLOCK        PD4
#define RADIO_CLOCK_DDR    DDRD
#define RADIO_CLOCK_PORT  PORTD
#define RADIO_DATA         PC3
#define RADIO_DATA_DDR     DDRC
#define RADIO_DATA_PORT  PORTC
#define RADIO_ACC 3 // PD3 = INT1
#endif



#if defined(__AVR_ATmega324__) || defined(__AVR_ATmega324P__) || defined(__AVR_ATmega324A__) || defined(__AVR_ATmega324PA__) || defined(__AVR_ATmega324PB__)
#define RADIO_COMMAND      PD6 //ICP 
#define RADIO_COMMAND_DDR  DDRD
#define RADIO_COMMAND_PORT  PORTD
#define RADIO_COMMAND_PIN PIND
#define RADIO_CLOCK        PB7
#define RADIO_CLOCK_DDR    DDRB
#define RADIO_CLOCK_PORT  PORTB
#define RADIO_DATA         PB5
#define RADIO_DATA_DDR     DDRB
#define RADIO_DATA_PORT  PORTB
#define RADIO_ACC 3
#endif


 bt module is connected to HW serial pins
